src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/miner/IndustrialMiner.java

```diff
@@ -52,6 +52,7 @@
     protected final List<MachineFuel> fuelTypes = new ArrayList<>();
 
     private final ItemSetting<Boolean> canMineAncientDebris = new ItemSetting<>(this, "can-mine-ancient-debris", false);
+    private final ItemSetting<Boolean> canMineDeepslateOres = new ItemSetting<>(this, "can-mine-deepslate-ores", false);
```

Comment:
Might be up for debate but I would reckon the default value for deepslate ores can be `true` here

Reply:
Yea, `true` might be the better choice, I can already see the bug reports for "My industrial miner doesn't work"